modules/docs/arrow-docs/docs/docs/arrow-incubator/typeclasses/monadtrans/README.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs-incubator
+title: MonadTrans
+permalink: /docs/arrow/mtl/typeclasses/monadtrans/
+redirect_from:
+  - /docs/typeclasses/monadtrans/
+---
+
+## MonadTrans
+
+MonadTrans is a typeclass that abstracts lifting arbitray monadic computations in another context.
+
+```kotlin:ank
+import arrow.mtl.extensions.optiont.monadTrans.monadTrans
+import arrow.core.extensions.id.monad.monad
+import arrow.core.extensions.*
+import arrow.core.*
+import arrow.mtl.*
+
+val result = OptionT.monadTrans().run {
+  Id.just("hello").lift(Id.monad())
+}
+```
```

Comment:
Here `ank` won't inline any text, could you please either remove the `val` or hint the type yourself?